components/Markdown.vue

```diff
@@ -79,18 +79,43 @@ export default {
   <Loading v-else />
 </template>
 
-<style lang="scss" scoped>
-  ::v-deep {
-    P {
-      font-size: initial;
-      line-height: initial;
-      font-weight: initial;
-      letter-spacing: initial;
-      font-style: normal;
+<style lang="scss">
+
+::v-deep {
+  P {
+    font-size: initial;
+    line-height: initial;
+    font-weight: initial;
+    letter-spacing: initial;
+    font-style: normal;
+  }
+  TH {
```

Comment:
Should the TH also be in `.markdown`? Interestingly I couldn't see either P or TH bleed in to other styles though

Reply:
I moved the TH under .markdown.

I also changed the line breaks. Before, the line would wrap in the middle of a word like `false`:


<img width="1136" alt="Screen Shot 2022-05-27 at 1 26 37 AM" src="https://user-images.githubusercontent.com/20599230/170664566-41c694ab-5036-4c4f-84a6-c731352d8e6b.png">

Now it doesn't wrap:
<img width="1143" alt="Screen Shot 2022-05-27 at 1 27 43 AM" src="https://user-images.githubusercontent.com/20599230/170664751-c4237361-30b2-457c-a711-08fbf184bab4.png">